### ISSUE TITLE
Cancel All Shows Wrong Count

### ISSUE THREAD

**Issue by JohnPTobe** [ngageoint/scale-ui#261]
The cancel all confirmation is hard coded to give the total running and queued jobs. If a user has filtered only the queued jobs then the number of reported canceled jobs is wrong.  The call to the api will correctly cancel only the queued jobs. We just need to fix the confirmation dialog to report the correct number if only running or only queued jobs are selected.

**Comment by JohnPTobe** [comment]
Similar problem with requeue all button. Also, queued jobs can be requeued.

**Comment by JohnPTobe** [comment]
https://github.com/ngageoint/scale-ui/blob/b48b03eb57538a64fec6b97a75d4ad38de6746ce/projects/developer/src/app/processing/jobs/component.ts#L331

**Comment by mikenholt** [comment]
@JohnPTobe  
> Also, queued jobs can be requeued.

Can you provide more info on this statement?

**Comment by JohnPTobe** [comment]
If you have a queued job you can requeue it.  Scale will cancel the queued job and make a new job and queue it. This was helpful for me in the past for fixing jobs that were stuck in the queued state because they were misconfigured for one reason or another.

**Comment by JohnPTobe** [comment]
It appears the ui only allows canceled/failed jobs to be requeued.

**Comment by cshamis** [comment]
Shouldn't it?

**Comment by JohnPTobe** [comment]
It's possible to requeue a queued job in the API. I've used this before to fix jobs that would never be scheduled, possibly due to resource issues or a misconfiguration.  You can work around this by canceling and then requeue but then it can be tricky to make sure you just requeue the ones you canceled and not other jobs that were canceled earlier.  It's probably something an advanced user would want to do and it's not part of the regular flow so we don't have to allow it in the ui. The main thing is we want to show the correct count for canceled/requeued jobs

**Comment by cshamis** [comment]
I agree.  Not something for regular users.  Probably not even anybody outside our own team. 

Yes.  Count needs fixed.  Ok.  I understand
